src/snowflake/cli/plugins/nativeapp/codegen/snowpark/python_processor.py

```diff
@@ -313,3 +315,63 @@ def get_src_py_file_to_dest_py_file_map(
         # TODO: Does this case 5 need to be considered?
 
         return src_py_file_to_dest_py_file_map
+
+
+def _create_new_sql_file_at_path(parent_dir: Path, file_name: str) -> Optional[Path]:
+    """
+    Creates an empty SQL file at the same location specified as the parent directory, with file_name as part of the new file's name.
+    """
+    sql_file_name = f"__{file_name}_{uuid.uuid4().hex}.sql"
```

Comment:
Not a fan of using uuid4 here for a few reasons. The most important one is that it won't be stable across bundle operations, so it will make any kind of diff unusable. The other is that UUID are terrible for humans. If I need a human to ever read a random ID, I much prefer to use base 36 random numbers. But really I think we should do something stable here, and avoid collisions in a more deterministic way.

Reply:
Made an update to the naming scheme, open for discussion. Thought about `GENERATED_BY_SNOWFLAKECLI` prefix or suffix, just like we use for comments in SQL DDL of application package and application. Open for discussion.

Reply:
Let's discuss tomorrow then.